### ISSUE TITLE
Model accuracy for logistic regression

### ISSUE THREAD

**Issue by ayushi-desai** [ayushi-desai/ayushi-lab2-2004#1]
It could be more if we change the parameters or the way of its implementation.

**Comment by ayushi-desai** [comment]
Hello
even I tried the same way you have implemented, however to increase the accuracy you can find the below code, it might help you:

import numpy as np
from sklearn.model_selection import train_test_split
from sklearn import linear_model
from sklearn.datasets import load_breast_cancer

#Load the Dataset of breast Cancer
cancer = load_breast_cancer()
# replacing text with numeric value
cancer.diagnosis = [1 if each == "M" else 0 for each in data.diagnosis]
# Fetching the input and ouput values
y = data.diagnosis.values
x_data = data.drop(['diagnosis'], axis = 1)
# Normalising values
x = (x_data - np.min(x_data))/(np.max(x_data) - np.min(x_data)).values
#splitting the training and testing data
x_train, x_test, y_train, y_test = train_test_split(
    x, y, test_size = 0.15, random_state = 42)
 
x_train = np.nan_to_num(x_train)
x_test = np.nan_to_num(x_test)
y_train = np.nan_to_num(y_train)
y_test = np.nan_to_num(y_test)
 
print("x train: ", x_train.shape)
print("x test: ", x_test.shape)
print("y train: ", y_train.shape)
print("y test: ", y_test.shape)

logreg = linear_model.LogisticRegression(random_state = 42, max_iter = 150)
print("train accuracy: {} ".format(
    logreg.fit(x_train, y_train).score(x_train, y_train)))
print("test accuracy: {} ".format(
    logreg.fit(x_train, y_train).score(x_test, y_test)))
